README.md

```diff
@@ -25,12 +25,11 @@ See the [live demo](https://onthegomap.github.io/planetiler-demo/) of vector til
 the [OpenStreetMap Americana Project](https://github.com/ZeLonewolf/openstreetmap-americana/).
 
 [![Planetiler Demo Screenshot](./diagrams/demo.png)](https://onthegomap.github.io/planetiler-demo/)
-Style [© OpenMapTiles](https://www.openmaptiles.org/)
-&middot; Data [© OpenStreetMap contributors](https://www.openstreetmap.org/copyright)
+[© OpenMapTiles](https://www.openmaptiles.org/) [© OpenStreetMap contributors](https://www.openstreetmap.org/copyright)
 
 ## Usage
 
-To generate a map of an area using the [basemap profile](planetiler-basemap), you will need:
+To generate a map of an area using the [OpenMapTiles profile](planetiler-openmaptiles), you will need:
```

Comment:
Link to the separate repo here since planetiler-openmaptiles directory doesn't exist